vpc.tf

```diff
@@ -3,18 +3,19 @@
  * Create VPC
  */
 module "vpc" {
-  source    = "github.com/silinternational/terraform-modules//aws/vpc?ref=8.8.0"
+  source  = "silinternational/vpc/aws"
+  version = "~> 1.0"
```

Comment:
How does this relate to the 8.8?

Reply:
We are gradually moving modules out of terraform-modules into their own repos. I don't recall exactly, but I believe [terraform-modules](https://github.com/silinternational/terraform-modules) 8.8.0 is very similar to [silinternational/vpc/aws](https://github.com/silinternational/terraform-aws-vpc) version 1.0.